core/src/toga/widgets/imageview.py

```diff
@@ -63,7 +70,7 @@ def rehint_imageview(image, style, scale=1):
 class ImageView(Widget):
     def __init__(
         self,
-        image: Image | Path | str | None = None,
+        image: Image | Path | str | Any | None = None,
```

Comment:
If you're using Any, there's no need to specify anything else, because *any* type is a subset of Any.